content/en/docs/zero-code/java/spring-boot-starter/annotations.md

```diff
@@ -43,6 +43,7 @@ public class TracedClass {
   @WithSpan(kind = SpanKind.CLIENT)
   public void tracedClientSpan() {}
 
+  @WithSpan
```

Comment:
I _think_ this block is autogenerated and should get fixed automatically now that https://github.com/open-telemetry/opentelemetry-java-examples/pull/827 has been merged, but I'm not sure

Reply:
Oh, I wasn't aware that a PR would be created automatically.
I'll wait for the PR to be opened.

By the way, could you let me know if there are any past PRs similar to what you pointed out?

https://github.com/open-telemetry/opentelemetry.io/pulls?page=1&q=is%3Apr+is%3Aclosed+Java+label%3Asig%3Ajava

Reply:
cc @zeitlinger